### ISSUE TITLE
Edit README.md seed nodes to be pasteable

### ISSUE THREAD

**Issue by theoreticalbts** [steemit/steem#295]
You should be able to copy-paste from README.md to config.ini.  Related #45 #289

**Comment by wmbutler** [comment]
```
seed-node = 45.55.217.111:12150
seed-node = 104.168.154.160:40696                    
seed-node = 52.4.250.181:39705                       
seed-node = 109.74.206.93:2001
seed-node = 45.114.118.146:2001
seed-node = 81.89.101.133:2001
seed-node = 40.76.37.6:2001
seed-node = 104.207.152.44:2201
seed-node = 52.38.66.234:2001
seed-node = 52.37.169.52:2001
seed-node = 52.26.78.244:2001
seed-node = 192.99.4.226:2001
seed-node = 46.252.27.1:1337
seed-node = steemd.pharesim.me:2001
seed-node = seed.steemed.net:2001
sseed-node = teem.clawmap.com:2001
seed-node = seed.steemwitness.com:2001
seed-node = steem-seed1.abit-more.com:2001
```

**Comment by abitmore** [comment]
@wmbutler typo.

**Comment by abitmore** [comment]
Just a note: I'm not using `config.ini` to configure multiple seed nodes due to this issue: https://github.com/cryptonomex/graphene/issues/648

**Comment by mvandeberg** [comment]
@abitmore, are you specifying your seed nodes in cli via script?

Also, I seem to recall that someone was running a site that would resolve to the fastest 20 seed nodes, but I cannot remember what it was...

**Comment by pmconrad** [comment]
If you mean http://seeds.quisquis.de/steem.html - I'm listing nodes by reliability, not by speed.
Also, the DNS seed is unreliable at the moment and not really recommended (it shouldn't hurt, but most of the time it only lists one IP, i. e. my own).

**Comment by abitmore** [comment]
> @abitmore, are you specifying your seed nodes in cli via script?

Right now, yes.

By the way, the node will crash when duplicate entries found in the seed node list, for example, one IP address come from `-s seed.steemed.com:2001` and another same address come from `-s steem.seeds.quisquis.de:2001`, I don't remember if it's fixed by #60.
